### PR TITLE
[NETBEANS-4499] Fixed unit tests for Symfony 2/3

### PR DESCRIPTION
**Pull request by KacerCZ**
https://issues.apache.org/jira/browse/NETBEANS-4499

- Created file `issue232490.xml` from `symfony2-commands.xml`
- Commented test SymfonyCommandsXmlParserTest:testIssue252901
- Enabled tests in Travis and Github Actions.

### CODE REVIEW

File: php/php.symfony2/test/unit/src/org/netbeans/modules/php/symfony2/commands/SymfonyCommandsXmlParserTest.java [hunk 1]
```diff
@@ -94,61 +96,57 @@ public void testIssue223639() throws Exception {
     }
 
     public void testIssue232490() throws Exception {
-        Reader reader = new BufferedReader(new FileReader(new File(getDataDir(), "issue232490.xml")));
+        Reader reader = new BufferedReader(new InputStreamReader(new FileInputStream(new File(getDataDir(), "issue232490.xml")), "UTF-8"));
```
Comment (tmysik) [inline php/php.symfony2/test/unit/src/org/netbeans/modules/php/symfony2/commands/SymfonyCommandsXmlParserTest.java:99]: A bit better way is `StandardCharsets.UTF_8[.name()]`.

**Comment by tmysik** [comment]
@KacerCZ 

> Commented test SymfonyCommandsXmlParserTest:testIssue252901

I am missing an explanation. Why is it done, please?

**Comment by KacerCZ** [comment]
@tmysik File for `SymfonyCommandsXmlParserTest:testIssue252901` was not donated and I don't know how to create it (what was changed).

This is related bug: https://bz.apache.org/netbeans/show_bug.cgi?id=252901

**Comment by tmysik** [comment]
@KacerCZ 

I can see it is very old issue so I guess if we can parse Symfony XML without errors now, we are OK. Thanks!

**Comment by tmysik** [comment]
@junichi11 Please merge it if it can be done now :) Thanks.

**Comment by junichi11** [comment]
Let's merge it after conflicting files are fixed. Thanks!

**Comment by KacerCZ** [comment]
@junichi11 Merge conflicts are solved.

**Comment by junichi11** [comment]
@KacerCZ Could you rebase and squash if possible?

**Comment by KacerCZ** [comment]
@junichi11 I tried, hopefuly it is OK.

**Pull request merged**